### PR TITLE
Update using-version-control-in-studio-pro.md

### PR DESCRIPTION
**Pull request by jacobboer**
Added a section to resolve an unintended version upgrade of the Mendix model "Reverting an accidental Studio Pro upgrade of your model"

Today is the 3rd time I had to help a colleague and couldnt find proper documentation. I hope this will help in future

### CODE REVIEW

File: content/refguide/using-version-control-in-studio-pro.md [hunk 1]
```diff
@@ -370,6 +370,25 @@ If you perform branching outside of Studio Pro, you will not be able to immediat
...
 
+### 9.5 Reverting an accidental Studio Pro upgrade of your model
+
+When working in different projects under different Studio Pro versions you may one day find your self with a model upgraded and commited to a newer Studio Pro version, while the the rest of the team is not yet ready to upgrade. 
+
+To undo this version upgrade of the model follow these steps:
+1. Inform your team, and prevent them from doing further commits.
+2. Close Studio Pro 
+3. In Windows Explorer go to your main project directy
+4. Right click the directory and open with SVN Show Log.
```
Comment (ironthorn) [inline content/refguide/using-version-control-in-studio-pro.md:381]: TortoiseSVN > Show log

File: content/refguide/using-version-control-in-studio-pro.md [hunk 2]
```diff
@@ -370,6 +370,25 @@ If you perform branching outside of Studio Pro, you will not be able to immediat
 
 To fix this, make a small commit on your branch in Studio Pro (for example, changing a documentation field). Studio Pro will then add the metadata that Mendix Cloud deployment requires, and you will be able to deploy your app.
 
+### 9.5 Reverting an accidental Studio Pro upgrade of your model
```
Comment (ironthorn) [inline content/refguide/using-version-control-in-studio-pro.md:373]: Thank you for taking the time to improve our documentation! This is a useful addition.

File: content/refguide/using-version-control-in-studio-pro.md [hunk 3]
```diff
@@ -370,6 +370,25 @@ If you perform branching outside of Studio Pro, you will not be able to immediat
...
+When working in different projects under different Studio Pro versions you may one day find your self with a model upgraded and commited to a newer Studio Pro version, while the the rest of the team is not yet ready to upgrade. 
+
+To undo this version upgrade of the model follow these steps:
+1. Inform your team, and prevent them from doing further commits.
+2. Close Studio Pro 
+3. In Windows Explorer go to your main project directy
+4. Right click the directory and open with SVN Show Log. 
+5. In SVN
+5.1 Select the latest version that was in the right Studio Pro version
+5.2 Right click that version and select "revert to this version", confirm and close with ok
```
Comment (ironthorn) [inline content/refguide/using-version-control-in-studio-pro.md:384]: Right click that *revision*

File: content/refguide/using-version-control-in-studio-pro.md [hunk 4]
```diff
@@ -370,6 +370,25 @@ If you perform branching outside of Studio Pro, you will not be able to immediat
...
+6 Right click on the folder again and now select "SVN Commit", enter your commit messageand click OK
```
Comment (ironthorn) [inline content/refguide/using-version-control-in-studio-pro.md:385]: TortoiseSVN > Commit and a space between "message" and "and"

File: content/refguide/using-version-control-in-studio-pro.md [hunk 5]
```diff
@@ -370,6 +370,25 @@ If you perform branching outside of Studio Pro, you will not be able to immediat
...
+7 After your commit has finished, remove or rename your main project folder
```
Comment (ironthorn) [inline content/refguide/using-version-control-in-studio-pro.md:386]: There is no need for this step for the person executing this process. The other team members, however, will need to do a fresh checkout OR run TortoiseSVN > Update on their project folder before opening the project in the desired version again.

**Review by Adam-Dupaski — APPROVED** [review]

File: content/refguide/using-version-control-in-studio-pro.md (hunk 2 above)
Comment (jacobboer) [reply content/refguide/using-version-control-in-studio-pro.md]: Thanks! The first time you run into this you tend to panic :-)

**Pull request merged**